stable/fluent-bit/values.yaml

```diff
@@ -16,6 +16,21 @@ backend:
   es:
     host: elasticsearch
     port: 9200
+    # Elastic Index Name
+    index: kubernetes_cluster
+    type: flb_type
+    logstash_prefix: kubernetes_cluster
+    # Optional username credential for Elastic X-Pack access
+    http_user: ""
+    # Password for user defined in HTTP_User
+    http_passwd: ""
+    tls: off
+    tls_verify: off
```

Comment:
Please add some documentation for these options, preferably make a start with the documentation in the README, as it's currently not very stable-worthy.

Reply:
README updated